### ISSUE TITLE
VS2015 Professional - Error when trying to open all Nuget options

### ISSUE THREAD

**Issue by ricardosmoke** [NuGet/Home#5125]
Hello. I'm facing a problem when I try to open NuGet in VS2015. When I try to open, I receive a error Unable to cast object of type 'NuGet.PackageManagement.VisualStudio.VSSettings' to type 'NuGet.Configuration.ISettings'. And after "Process cannot access the file because it is being used by another process". I can't upgrade Nuget in VS2015 and manually too. I reinstalled VS2015, but the problem continues. My version is VS2015 Professional. Please, help me. Thanks.

NuGet product use: ALL
NuGet version: I think that is 3.4.0.798
dotnet.exe --version: 1.0.3
VS Version: VS2015 Professional: 14.0.25431.01 Update 3
OS version (i.e. win10 v1607 (14393.321)):  Windows 7 Enterprise Service Pack 1

Worked before? If so, with which NuGet version: I think that only worked when I has community version, but I don't remember.

## Detailed repro steps so we can see the same problem
Only need to click in any option from nuget package manager.

THanks.

**Comment by zhili1208** [comment]
could you try to uninstall nuget vsix and install 3.5 vsix from https://dist.nuget.org/index.html?

**Comment by ricardosmoke** [comment]
I tried already, but ever I receive the error that the "process cannot access the file because it is being used by another process". I can't update or uninstall. Ever the same error. Thanks!

**Comment by zhili1208** [comment]
Could you try following?

1. Open visual studio and don't open any solution or project.
2. Open package manager console from tools/NuGet Package Manager.

does this works? if not, delete NuGet.Config from %appdata%\NuGet and try again

**Comment by ricardosmoke** [comment]
Yay! I solved my problem trying what is explained here, in paulhickman-a365's comment:

https://github.com/NuGet/Home/issues/1138

"I had the issue on two machines. On the simplest scenario it was a clean build, with Windows 2012 R2, Visual Studio 2015, SQL Server 2014, iTunes, Chrome and Notepad++ as the only applications installed. Attempting to upgrade NuGet was the first thing I did after installing visual studio. There was no antivirus or similar installed on the server. My log file looks the same as that posted above.

If I close visual studio and delete all the folders in C:\Program Files (x86)\Microsoft Visual Studio 14.0\Common7\IDE\Extensions that contain a copy of Nuget, I don't get any errors about files being in use while deleting.

I can then start visual studio and download and install Nuget from the Extensions gallery successfully."

-=-=-=-=-=

The only difference is that I downloaded nuget from the official website and installed. Now all works fine!

Thanks a lot!
